testing/ponysay/APKBUILD

```diff
@@ -0,0 +1,22 @@
+# Maintainer: Matthew T Hoare <matthew.t.hoare@gmail.com>
+# Contributor: Sven-Hendrik Haase <sh@lutzhaase.com>
+# Contributor: Erkin Batu Altunbaş <erkinbatu@gmail.com>
+
+pkgname=ponysay
+pkgver=3.0.2
+pkgrel=0
+pkgdesc="cowsay reimplemention for ponies"
+url="http://erkin.github.com/ponysay/"
+arch="all"
+license="GPL3"
+depends="python bash"
+makedepends="git"
+subpackages="$pkgname-doc"
+source="ponysay-$pkgver.tar.gz::https://github.com/erkin/ponysay/tarball/$pkgver"
+md5sums="84b04705a0a57b2f43327ac6b7952013 ponysay-$pkgver.tar.gz"
```

Comment:
The MD5 sum does not seem to match, and nowadays we are using sha256sums instead of md5.

Reply:
OK, I will update.

Reply:
Please use `abuild checksum` to generate and insert checksums into APKBUILD.